templates/quickstart-confluence-master.template.yaml

```diff
@@ -1212,7 +1223,7 @@ Resources:
             VolumeSize: !Ref ClusterNodeVolumeSize
         - DeviceName: /dev/xvdf
           NoDevice: true
-      KeyName: !If [ KeyProvided, !Ref KeyPairName, !ImportValue ATL-DefaultKey ]
```

Comment:
Did we validate that this is a feature we don't want to utilise in the quickstart anymore? Do we know why we wanted to have automatically imported keys in the first place 🤔

Reply:
I think it is the same thing as I raised in https://github.com/atlassian/quickstart-atlassian-jira/pull/26#discussion_r324510455

We are now making it mandatory to provide a KeyPair and don't import it from ASI stack. That will kind of improve experience on the form for the -non-vpc template but overall we now make the argument mandatory. Thinking about it more, doesn't this potentially break the deployment?

Let's say I have created `ASI` stack with `keypair-A`, later I am creating Confluence stack and from the dropdown I select `keypair-B`. The instances are created but when something requires investigation on the cluster nodes I need to login on the Bastion host with `keypair-A` but I cannot use this key pair to log in to cluster nodes because that one expects `keypair-B`. I can use Session manager or I can add both keys to my ssh keystore and use key forwarding with `ssh -A`.